Formula/h/hyfetch.rb

```diff
@@ -0,0 +1,53 @@
+class Hyfetch < Formula
+  include Language::Python::Virtualenv
+
+  desc "Neofetch with LGBTQ+ pride flags + maintained neofetch"
+  homepage "https://github.com/hykilpikonna/hyfetch"
+  url "https://files.pythonhosted.org/packages/bf/04/13a5091a1da014fad160710abfad2aa03a72bc41e4678c95be2b5ee67818/HyFetch-1.4.10.tar.gz"
+  sha256 "023733fa358380fd41589cd80e9b008d376eeef16b489fba8ee8610e71e42057"
+  license "MIT"
+  head "https://github.com/hykilpikonna/hyfetch.git", branch: "master"
+  
+  depends_on "python@3.11"
+
+  on_macos do
+    depends_on "screenresolution"
+  end
+
+  conflicts_with "neofetch", because: "conflicting name"
```

Comment:
Not a big thing, but I preferred the wording of `because: "both install `neofetch` binary"` as in [this review](https://github.com/Homebrew/homebrew-core/pull/139819#discussion_r1300857290).

And I think the `conflicts_with` also needs to be added to the `neofetch` formula.